src/lib/Components/Inbox/Bids/ActiveBid.tsx

```diff
@@ -134,32 +134,13 @@ export default Relay.createContainer(ActiveBid, {
       fragment on LotStanding {
         is_leading_bidder
         active_bid {
-          id
           max_bid {
-            cents
             display
           }
           sale_artwork {
-            lot_label
             lot_number
-            position
             reserve_status
-            counts {
-              bidder_positions
-            }
-            sale {
-              live_start_at
-              end_at
-              is_live_open
-              is_closed
-            }
-            highest_bid {
-              cents
-              display
-            }
             artwork {
-              id
-              title
```

Comment:
In this particular case, it was in part because the dev process involved a lot of trial and error to figure out where the display data came from. 

Also, since there are still some unknowns about active bids, namely where rows should be routing to (artwork view, auction page, etc...), we decided to hold on for that because the auctions team was changing auction urls. The plan was to update the fragment after we figure the new URI format. Anyway, sorry I should've left a comment explaining that.